taipy/gui/viselements.json

```diff
@@ -761,6 +761,31 @@
         ]
       }
     ],
+    ["login", {
+        "inherits": ["shared"],
+        "properties": [
+            {
+                "name": "title",
+                "default_property": true,
+                "type": "str",
+                "default_value": "Log-in",
+                "doc": "The title of the login dialog."
+            },
+            {
+                "name": "on_action",
+                "type": "Callback",
+                "default_value": "on_login",
+                "doc": "The name of the function that is triggered when the dialog's button is pressed.<br/><br/>All the parameters of that function are optional:\n<ul>\n<li>state (<code>State^</code>): the state instance.</li>\n<li>id (str): the identifier of the button.</li>\n<li>payload (dict): the details on this callback's invocation.<br/>\nThis dictionary has the following keys:\n<ul>\n<li>action: the name of the action that triggered this callback.</li>\n<li>args: List(user, password, current page).</li>\n</ul>\n</li>\n</ul>",
```

Comment:
Is the id the id of the button, or the login control? Wouldn't it make more sense?

Reply:
```to be unformatted...
                <li>args: a tuple with three elements:
                    <ul>
                      <li>The first element is the user name</li>
                      <li>The second element is the password</li>
                      <li>The third element is the current page name</li>
                   </ul>
                 </li>
```